<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYBaselineCorrectionCurveDockGeneralTab</class>
 <widget class="QWidget" name="XYBaselineCorrectionCurveDockGeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>939</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="20" column="2" colspan="3">
    <widget class="QComboBox" name="cbPlotRanges"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lDataSourceCurve">
     <property name="text">
      <string>Curve:</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lXColumn">
     <property name="text">
      <string>X-Data:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="TimedLineEdit" name="leName"/>
   </item>
   <item row="16" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lDataSourceType">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="3">
    <widget class="QFrame" name="frameRange">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lMaxDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lMinDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="UTCDateTimeEdit" name="dateTimeEditMin">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UTCDateTimeEdit" name="dateTimeEditMax">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leMin"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leMax"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lSubtraction">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Subtraction</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lData">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="lPlotRange">
     <property name="text">
      <string>Plot Range:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lARPLSSmoothness">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Smoothness (Power of 10):</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QCheckBox" name="cbAutoRange">
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QCheckBox" name="chkLegendVisible">
     <property name="text">
      <string>Legend</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lARPLSTerminationRatio">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Termination Ratio:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="3">
    <widget class="QSpinBox" name="sbARPLSSmoothness">
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <spacer name="verticalSpacerGeneral">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lXRange">
     <property name="text">
      <string>X-Range:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="2" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lResults">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Results:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QCheckBox" name="chkVisible">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="3">
    <widget class="QComboBox" name="cbMethod"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lYColumn">
     <property name="text">
      <string>Y-Data:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="ResizableTextEdit" name="teComment">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="19" column="4">
    <widget class="QPushButton" name="pbRecalculate">
     <property name="text">
      <string>Recalculate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QComboBox" name="cbDataSourceType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lARPLSIterations">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Iterations:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lMethod">
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2" colspan="3">
    <widget class="QTextEdit" name="teResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="3">
    <widget class="QSpinBox" name="sbARPLSIterations">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="3">
    <widget class="NumberSpinBox" name="sbARPLSTerminationRatio">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>Spinbox for numbers.</string>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="feedback" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>frontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>frontend/widgets/TimedLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>UTCDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>UTCDateTimeEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>cbDataSourceType</tabstop>
  <tabstop>cbAutoRange</tabstop>
  <tabstop>leMin</tabstop>
  <tabstop>leMax</tabstop>
  <tabstop>dateTimeEditMin</tabstop>
  <tabstop>dateTimeEditMax</tabstop>
  <tabstop>cbMethod</tabstop>
  <tabstop>teResult</tabstop>
  <tabstop>pbRecalculate</tabstop>
  <tabstop>cbPlotRanges</tabstop>
  <tabstop>chkLegendVisible</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
